client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -324,6 +332,25 @@ private static KeyStore loadTrustStore(File trustStorePath, Optional<String> tru
         return trustStore;
     }
 
+    private static KeyStore loadSystemKeyStore(Optional<String> keyStoreType)
+            throws IOException, GeneralSecurityException
+    {
+        String osName = Optional.ofNullable(StandardSystemProperty.OS_NAME.value()).orElse("");
```

Comment:
Extract this into a method, it's used in two places now.

Reply:
I've refactored the two system store methods to reduce the duplication.